examples/image_classification/blog_test.py

```diff
@@ -0,0 +1,52 @@
+import os
+import autogluon as ag
+from autogluon import ImageClassification as task
+from mxnet import optimizer as optim
+
+def task_dog_breed_identification():
+    # data_loader
+    data_path = '/home/ubuntu/workspace/dataset'
+    dataset = 'dog-breed-identification_mini'
+    dataset_path = os.path.join(data_path, dataset, 'train')
+    test_path = os.path.join(data_path, dataset, 'test')
+    csv_path = os.path.join(data_path, dataset, 'sample_submission.csv')
+    load_dataset = task.Dataset(dataset_path)
+
+    # images_path = os.path.join(data_path, dataset, 'images')
+    # label_path =  os.path.join(data_path, dataset, 'labels.csv')
+    # load_dataset = task.Dataset(images_path, label_file=label_path) # bug
+
+    # search space
+    @ag.obj(
+        learning_rate=ag.space.Real(0.3, 0.5),
+        momentum=ag.space.Real(0.90, 0.95),
+        wd=ag.space.Real(1e-6, 1e-4, log=True),
+        multi_precision=False
+    )
+    class NAG(optim.NAG):
+        pass
+
+    classifier = task.fit(dataset=load_dataset,
+                          # pretrain model
+                          # nets=ag.Categorical('resnext_101_standforddog', 'resnet151_v1d_standford'),
```

Comment:
`standford_dog_resnet152_v1` and `standford_dog_resnext101_64x4d` are in PR https://github.com/awslabs/autogluon/pull/236